Assets/__Scripts/UI/LogLineUI.cs

```diff
@@ -0,0 +1,149 @@
+            Heading("System information:", true) +
+            GenerateSystemInfo() + 
+
+            Heading("Exception:") +
+            _logline.Message + "\n" + _logline.StackTrace +
+
+            Heading("Recent log messages before error:") + 
+            _previousMessages,
+            "ChroMapper " + Application.version + " bug report info"
+        );
+        ReportButton.image.color = Color.green;
+    }
+
+    private static IEnumerator WriteErrorToFile(string text)
+    {
+        File.WriteAllText("error.txt", text);
+        yield break;
+    }
+
+    private static IEnumerator UploadToPasteee(string text, string title = "Untitled")
+    {
+        var requestBody = new JSONObject();
+        requestBody["key"] = PasteeeKey;
+        requestBody["description"] = title;
+
+        var sections = new JSONArray();
+        var section = new JSONObject();
+        section["name"] = "Main";
+        section["contents"] = text;
+        
+        sections.Add(section);
+        requestBody["sections"] = sections;
+
+        using (var www = UnityWebRequest.Post("https://api.paste.ee/v1/pastes", ""))
+        {
+            www.uploadHandler = new UploadHandlerRaw(Encoding.UTF8.GetBytes( requestBody.ToString()))
+            {
+                contentType = "application/json"
+            };
+            yield return www.SendWebRequest();
+            var result = JSON.Parse(www.downloadHandler.text);
+
+            if (result.HasKey("link"))
+            {
+                Application.OpenURL(result["link"]);
+            }
+            else
+            {
+                Debug.LogError("Failed to upload bug report!");
+            }
+        }
+    }
+
+    private static IEnumerator UploadToPastebin(string text, string title = "Untitled")
+    {
+        var form = new WWWForm();
+        form.AddField("api_dev_key", DevKey);
+        form.AddField("api_option", "paste");
+        form.AddField("api_paste_code", text);
+        form.AddField("api_paste_name", title);
+        form.AddField("api_paste_format", "csharp");
+        form.AddField("api_paste_private", 1); // Unlisted
+        form.AddField("api_paste_expire_date", "N"); // Never
+
+        using (var www = UnityWebRequest.Post("https://pastebin.com/api/api_post.php", form))
+        {
+            yield return www.SendWebRequest();
+            var result = www.downloadHandler.text;
+
+            if (result.Contains("Bad API request"))
+            {
+                Debug.LogError("Failed to upload bug report! " + result);
+            }
+            else
+            {
+                Application.OpenURL(result);
+            }
+        }
+    }
+    
+    private static IEnumerator CreateAsync(string text, string title = "Untitled", string language = "csharp", int visibility = 1, string expiration = "N")
+    {
+        //yield return WriteErrorToFile(text);
+        yield return UploadToPasteee(text, title);
+        //yield return UploadToPastebin(text, title);
```

Comment:
Would be pretty gamer if we had a dialog box that allowed the user to pick between the available options (although if this happens I dont think the coroutine would be necessary)

Reply:
Guess I removed the comment but I moved from pastebin to pasteee because pastebin both auto-deleted my initial tests and limits you to 25 uploads per day (I assume per ip and not per key) but I left the code here "just in case".

I could allow you to pick a file instead but I'm not sure there's really a downside to always uploading it.